app/views/spaces/statistics.scala.html

```diff
@@ -1,11 +1,37 @@
-@(space: ProjectSpace, classes: String)(implicit user: Option[models.User])
+@(space: ProjectSpace, spaceBytes: Long, classes: String, isPublic: Option[Boolean], spaceFiles: Integer)(implicit user: Option[models.User])
 
 @import play.api.i18n.Messages
+@import _root_.util.Formatters._
 <div class="@classes">
-    <div class="col-md-12 col-lg-12 col-sm-12">
-        <h3>Statistics</h3>
-        <p><span class="glyphicon glyphicon-user"></span> Members: @space.userCount</p>
-        <p><span class="glyphicon glyphicon-th-list"></span> Collections: @space.collectionCount</p>
-        <p><span class="glyphicon glyphicon-briefcase"></span> Datasets: @space.datasetCount</p>
+    <div class="col-md-12" id="resources-panel-container">
+        <div class="panel panel-default" id="resources-panel" data-clampedwidth=".col-md-5">
+            <div class="panel-heading">
+                <h3 class="panel-title">Resources</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-group">
+
+@*                    <li class="list-group-item" title="@humanReadableNumber(spacesCount) @Messages("spaces.title")">*@
```

Comment:
if not used, please remove commented out code.

Reply:
This is now removed.